docker/build.sh

```diff
@@ -35,14 +35,6 @@ else
     image_name_suffix="-cuda"
 fi
 
-# Set prebuilt options
-if [ "$option_no_prebuilt" = "true" ]; then
-    targets="devel"
-else
-    # default targets include devel and prebuilt
-    targets=""
```

Comment:
It's from https://github.com/autowarefoundation/autoware/pull/3253.
I'm sorry for my insufficient review in the PR.

However, the changes by this PR look a little redundant.
Is it impossible to write as follows instead?

```
targets="devel,prebuilt"
targets="*"
```

Reply:
I got it. I'll revert it.
But unfortunately, I think your solution might not work.
If the targets are passed chained like `devel target`, them the `bake` will process the line as-is and the command will fail finally.
According to https://docs.docker.com/engine/reference/commandline/buildx_bake/#file , when passing multiple targets to the `bake`, we have to specify the name(s) with space-separated manner... In scripting mode, targets should not be a single string but array-like.